src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorTestConstants.java

```diff
@@ -11,54 +11,56 @@
+    public static final String HG19_2_REFERENCE_FILE_NAME = FUNCOTATOR_LARGE_FILES_DIR + "b37.2.tar.gz";
 
     public static final String REFERENCE_VERSION_HG19 = "hg19";
     public static final String REFERENCE_VERSION_HG38 = "hg38";
 
+    // MUC16 info:
     public static final SimpleInterval MUC16_POSITION = new SimpleInterval("chr19", 8959520, 9092018);
-    public static final String MUC16_GENCODE_ANNOTATIONS_FILE_NAME = FUNCOTATOR_DATA_SOURCES_MAIN_FOLDER + "gencode_muc16" + File.separator + "hg19" + File.separator + "gencode.v19.MUC16.gtf";
-    public static final String MUC16_GENCODE_TRANSCRIPT_FASTA_FILE = FUNCOTATOR_DATA_SOURCES_MAIN_FOLDER + "gencode_muc16" + File.separator + "hg19" + File.separator + "gencode.v19.MUC16_transcript.fasta";
     public static final String MUC16_TRANSCRIPT = "ENST00000397910.4";
     public static final String MUC16_PATHOLOGICAL_TRANSCRIPT = "ENST00000599436.1";
     public static final String MUC16_GENCODE_NON_BASIC_ANNOTATIONS_FILE_NAME = FUNCOTATOR_TEST_DIR + "gencode.v19.MUC16.non-basic.gtf";
 
+    // PIK3CA info:
     public static final SimpleInterval PIK3CA_POSITION = new SimpleInterval("chr3", 178866311, 178957881);
-    public static final String PIK3CA_GENCODE_ANNOTATIONS_FILE_NAME = FUNCOTATOR_DATA_SOURCES_MAIN_FOLDER + "gencode_pik3ca" + File.separator + "hg19" + File.separator + "gencode.v19.PIK3CA.gtf";
-    public static final String PIK3CA_GENCODE_TRANSCRIPT_FASTA_FILE = FUNCOTATOR_DATA_SOURCES_MAIN_FOLDER + "gencode_pik3ca" + File.separator + "hg19" + File.separator + "gencode.v19.PIK3CA_transcript.fasta";
     public static final String PIK3CA_TRANSCRIPT = "ENST00000263967.3";
 
-    public static final String GTF_CHR3_FILE_NAME = GATKBaseTest.largeFileTestDir + "funcotator" + File.separator + "gencode.v19.chr_patch_hapl_scaff.chr3.gtf";
-
-    public static final String GENCODE_TRANSCRIPT_FASTA_FILE_NAME = GATKBaseTest.largeFileTestDir + "funcotator" + File.separator + "gencode.v19.pc_transcripts.fasta";
+    public static final String GTF_CHR3_FILE_NAME = FUNCOTATOR_LARGE_FILES_DIR + "gencode.v19.chr_patch_hapl_scaff.chr3.gtf";
 
     public static final String VARIANT_FILE_HG19_CHR3 = FUNCOTATOR_TEST_DIR + "snpTest_chr3_hg19.vcf";
     public static final String VARIANT_FILE_HG19_CHR19 = FUNCOTATOR_TEST_DIR + "snpTest_chr19_hg19.vcf";
 
     // ----------------------------------------------------------------------
     // Data source variables:
 
+    public static final String DUMMY_DATA_SOURCES_TAR_GZ             = FUNCOTATOR_LARGE_FILES_DIR + "dummyDataSources.tar.gz";
+    public static final String DUMMY_DATA_SOURCES_TAR_GZ_SHA256_FILE = FUNCOTATOR_LARGE_FILES_DIR + "dummyDataSources.sha256";
+    public static final String DUMMY_DATA_SOURCES_FOLDER             = FUNCOTATOR_LARGE_FILES_DIR + "dummyDataSources";
+
+    public static final String FUNCOTATOR_DATA_SOURCES_MAIN_FOLDER = FUNCOTATOR_LARGE_FILES_DIR + "funcotator_dataSources" + File.separator;
```

Comment:
Should this (and related declarations below) be moved to the top of the class, with a comment explaining what it contains (since we should be using this for most of our future tests, right?)

Reply:
Sure.  Fixed.